gcp/modules/couchdb/resources/couchdb-network-policy.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: couchdb-network-policy
+  namespace: gpii
+spec:
+  podSelector:
+    matchLabels:
+      app: couchdb
+  policyTypes:
+  - Ingress
+  - Egress
+  ingress:
+  - from:
+    - ipBlock:
+        cidr: 10.0.0.0/24
```

Comment:
If I'm reading the cluster setup correctly, it'll be guaranteed as long as this doesn't substantially change:

https://github.com/gpii-ops/gpii-infra/blob/master/gcp/modules/gke-network/main.tf#L27

Is it reasonable to expect this to *ever* change?

Reply:
Nope, it's not - depends on type of cluster netowork and config, but see https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1beta1/projects.zones.clusters for details.

In general, we should not assign policies based on IP addresses but rather based on labels.

Reply:
@stepanstipl - is there any value to using ipBlock(s) in our network policies?

Reply:
I think unless real reason our policies should all be label based rather than ip based. Also what is the intention here - to allow access from what exactly?

Reply:
To clarify - the rules says traffic is allowed from sources that:
- have source IP in `cidr: 10.0.0.0/24`
**OR**
- are pods with label `app: couchdb`.

I guess you were aiming for AND logic - all pods with label and IPs in that range.